Lib/test/test_pyrepl/test_unix_console.py

```diff
@@ -303,3 +303,13 @@ def test_getheightwidth_with_invalid_environ(self, _os_write):
             self.assertIsInstance(console.getheightwidth(), tuple)
             os.environ = []
             self.assertIsInstance(console.getheightwidth(), tuple)
+
+    @unittest.skipUnless(sys.platform == "darwin",  "requires OS X")
```

Comment:
```suggestion
    @unittest.skipUnless(sys.platform == "darwin", "requires OS X")
```

Reply:
interesting when lint not found this?

Reply:
The lint job is very simple because the code base is too old and has too many coding styles =/